docusaurus.config.js

```diff
@@ -35,7 +36,24 @@ module.exports = {
     footer: {
       style: 'dark',
       copyright: `Made with ❤ by the humans at <a href='https://mvpworkshop.co'>MVP Workshop</a>`
-    }
+    },
+    typesense: {
+      typesenseCollectionName: 'test1-doc_1641038942', // Replace with your own doc site's name. Should match the collection name in the scraper settings.
+      
+      typesenseServerConfig: {
+        nodes: [
+          {
+            host: 'fvxcy792qoz86bwtp-1.a1.typesense.net',
+            port: 443,
+            protocol: 'https',
+          },
+        ],
+        apiKey: '5TLdeqNihrV4sw6afUssvYYPrRdQGucA',
```

Comment:
Can we please regenerate this key and use it as a secret here instead of just pasting it publicly?

cc @ZeljkoBenovic

Reply:
There should be a way to set this up as env var: https://www.netlify.com/blog/2021/07/05/easy-access-environment-variables